### PR TITLE
Fix #3777 - reformat as table

### LINKED ISSUE #3777: Names of parameters could be in bold
The actual parameter names being documented could stand to be in bold
Example:
**Append** Data type: SwitchParameter

This applies to all of the pages linked to from https://docs.microsoft.com/en-us/powershell/developer/cmdlet/standard-cmdlet-parameter-names-and-types

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 971f2aef-be6b-3b2e-cfe4-71f12468ad44
* Version Independent ID: cbb8fe0b-8b8a-5910-9854-e7b8c3a7e029
* Content: [Activity Parameters](https://docs.microsoft.com/en-us/powershell/developer/cmdlet/activity-parameters)
* Content Source: [developer/cmdlet/activity-parameters.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/developer/cmdlet/activity-parameters.md)
* Product: **powershell**
* GitHub Login: @joeyaiello
* Microsoft Alias: **jaiello**

### PR DESCRIPTION
**Pull request by sdwheeler**
The original articles in MSDN were formatted as a table. Recreated the tables here.

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9231b07](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/9231b07180fc559e3e993927a68fcdf979534bd9)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9231b07](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/9231b07180fc559e3e993927a68fcdf979534bd9)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 48 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9231b07](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/9231b07180fc559e3e993927a68fcdf979534bd9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/cmdlet/activity-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/activity-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/activity-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/date-and-time-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/date-and-time-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/date-and-time-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/format-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/format-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/format-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/property-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/property-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/property-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/quantity-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/quantity-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/quantity-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/resource-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/resource-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/resource-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/security-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/security-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/security-parameters?branch=pr-en-us-3779) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C27%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902271856310911-3779%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eVOSaJ1JJD9%2F4tv2br%2BaTMlQ8TqjUQ6EvogXQvg0jbg%3D&st=2019-02-27T19%3A05%3A59Z&se=2019-03-30T19%3A10%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6958eec](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/6958eecefe6be02bc2dd63779ea0ffb5f3400771)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[6958eec](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/6958eecefe6be02bc2dd63779ea0ffb5f3400771)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 48 sec(s)_

File: developer/cmdlet/activity-parameters.md [hunk 1]
```diff
@@ -13,215 +13,50 @@ caps.latest.revision: 5
...
+|**Stream**<br>Data type: SwitchParameter|Implement this parameter so the user can stream multiple output objects through the pipeline when the parameter is specified.|
+|**Strict**<br>Data type: SwitchParameter|Implement this parameter so that all errors are handled as terminating errors when the parameter is specified.|
+|**TempLocation**<br>Data type: String|Implement this parameter so the user can specify the location of temporary data that is used during the operation of the cmdlet.|
+|**Timeout**<br>Data type: Int32|Implement this parameter so that the user can specify the timeout interval (in milliseconds).|
+|**Truncate**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will truncate its actions when the parameter is specified. If the parameter is not specified, the cmdlet performs another action.|
+|**Verify**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will test to determine whether an action has occurred when the parameter is specified.|
+|**Wait**<br>Data type: SwitchParameter|Implement this parameter so that the cmdlet will wait for user input before continuing when the|parameter is specified.
```
Comment (DCtheGeek) [inline developer/cmdlet/activity-parameters.md]: There's an extra `|` in the line causing the table to have a 3rd column and some content in that column.

Comment (sdwheeler) [reply developer/cmdlet/activity-parameters.md]: Fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[444f1f9](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/444f1f9ef8789bcb5614c5f5783cb47cf0a6933a)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 9 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[444f1f9](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/444f1f9ef8789bcb5614c5f5783cb47cf0a6933a)_: 

:clock10: **Incremental building**: average incremental building time is _12 min(s) 49 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[444f1f9](https://github.com/sdwheeler/PowerShell-Docs/pull/3779/commits/444f1f9ef8789bcb5614c5f5783cb47cf0a6933a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[developer/cmdlet/activity-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/activity-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/activity-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/date-and-time-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/date-and-time-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/date-and-time-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/format-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/format-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/format-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/property-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/property-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/property-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/quantity-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/quantity-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/quantity-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/resource-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/resource-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/resource-parameters?branch=pr-en-us-3779) |
[developer/cmdlet/security-parameters.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-issue3777/developer/cmdlet/security-parameters.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/developer/cmdlet/security-parameters?branch=pr-en-us-3779) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C27%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201902272038545250-3779%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9HwDCyiaZx%2BiGwlz2mizpS1tavlDGBwisDCPfIOl9rY%3D&st=2019-02-27T20%3A47%3A56Z&se=2019-03-30T20%3A52%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**